aip-console-tools-core/pom.xml

```diff
@@ -50,9 +50,21 @@
         <dependency>
             <groupId>com.squareup.okhttp3</groupId>
             <artifactId>okhttp</artifactId>
-            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jetbrains.kotlin</groupId>
+                    <artifactId>kotlin-stdlib</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.jetbrains.kotlin</groupId>
+                    <artifactId>kotlin-stdlib-jdk7</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.jetbrains.kotlin</groupId>
+                    <artifactId>kotlin-stdlib-jdk8</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Why did you add those exclusions ? Do they cause issue during the build with enforcer ? I don't see these as dependencies in the okhttp pom; I only see "android" but it is a "provided" scope meaning it shouldn't be added.

Can you try removing them and seeing if you are having any issue ? If you want we can have a call to discuss this.

Reply:
compile complaint for version clash

Reply:
That's weird, but yeah if it causes errors with the enforcer plugin, then keep those around. I'll have a check later.

Reply:
sounds well and seems to work with new versions